src/NUnitFramework/tests/Constraints/RangeConstraintTests.cs

```diff
@@ -87,5 +87,77 @@ public void ShouldThrowExceptionIfFromIsLessThanTo()
         {
             Assert.Throws<ArgumentException>(() => new RangeConstraint( 42, 5 ));
         }
+
+        #if !NETCF && !SILVERLIGHT  && !PORTABLE
```

Comment:
Can you remove the NETCF and SILVERLIGHT conditionals, they have been removed from the main solution now.